### PR TITLE
Change author

### PR DESCRIPTION
**Pull request by NicktheGeek**
@reaktivstudios/reviewers this is ready to review

### CODE REVIEW

File: gppro-google-webfonts.php [hunk 1]
```diff
@@ -964,6 +964,9 @@ public function google_stack_list( $stacks = array() ) {
 
 				// If we dont already have the font, add it.
 				if ( ! isset( $stacks[ $type ][ $family ] ) ) {
+					if ( empty( $stacklist[ $family ] ) ) {
```
Comment (jjeaton) [inline gppro-google-webfonts.php:967]: Is this an unrelated change?

**Review by jjeaton — APPROVED** [review]

File: gppro-google-webfonts.php (hunk 1 above)
Comment (NicktheGeek) [reply gppro-google-webfonts.php:967]: huh, I didn't change that. I thought I made sure to start this from master, I wonder if this is one of those weird issues with git showing something already in the destination.
